### ISSUE TITLE
Problem when library unintentionally tries to mock `hasOwnProperty`

### ISSUE THREAD

**Issue by insightfuls** [NagRock/ts-mockito#107]
This problem is similar to #28. `ts-mockito` tries to mock `hasOwnProperty` and fails with this stack trace:

```
    TypeError: methodStub.hasMatchingInAnyGroup is not a function
        at Mocker.getMethodStub (node_modules/ts-mockito/lib/Mock.js:186:38)
        at Object.hasOwnProperty (node_modules/ts-mockito/lib/Mock.js:118:36)
        at Mocker.createInstanceActionListener (node_modules/ts-mockito/lib/Mock.js:104:27)
        at node_modules/ts-mockito/lib/Mock.js:88:27
        at Array.forEach (<anonymous>)
        at node_modules/ts-mockito/lib/Mock.js:79:66
        at Array.forEach (<anonymous>)
        at Mocker.processProperties (node_modules/ts-mockito/lib/Mock.js:78:58)
        at new Mocker (node_modules/ts-mockito/lib/Mock.js:27:18)
        at Object.mock (node_modules/ts-mockito/lib/ts-mockito.js:25:12)
```

I encountered this while trying to mock some built-in objects in NW.js. One way to reproduce it in that environment is to:
1. create a blank NW.js app (an empty HTML page is fine)
2. use `npm` to install `ts-mockito`
3. start the NW.js app using the SDK/development version of NW.js
4. right-click the page and choose `Inspect` which will open the Chromium Devtools
5. in the console in the DevTools, execute:
```
mockito = require("ts-mockito");
mockito.mock(nw.Menu);
```

**Comment by kierans** [comment]
👍

**Comment by NagRock** [comment]
Fixed in 2.3.1
https://github.com/NagRock/ts-mockito/releases/tag/2.3.1

Thanks @insightfuls for contributing!
